modules/engines/engine-core/src/main/java/com/sap/xsk/hdb/ds/api/IXSKDataStructureModel.java

```diff
@@ -13,101 +13,55 @@
 
 public interface IXSKDataStructureModel {
 
-    /**
-     * File extension for *.hdbtable files
-     */
-    public static final String FILE_EXTENSION_TABLE = ".hdbtable";
-    /**
-     * File extension for *.hdbview files
-     */
-    public static final String FILE_EXTENSION_VIEW = ".hdbview";
-    /**
-     * File extension for *.hdbdd files
-     */
-    public static final String FILE_EXTENSION_ENTITIES = ".hdbdd";
-    /**
-     * File extension for *.calculationview files
-     */
-    public static final String FILE_EXTENSION_CALCULATION_VIEW = ".calculationview";
-    /**
-     * File extension for *.hdbcalculationview files
-     */
-    public static final String FILE_EXTENSION_HDBCALCULATION_VIEW = ".hdbcalculationview";
-    /**
-     * File extension for *.hdbprocedure files
-     */
-    public static final String FILE_EXTENSION_HDBPROCEDURE = ".hdbprocedure";
-    /**
-     * File extension for *.hdbtablefunction files
-     */
-    public static final String FILE_EXTENSION_HDBTABLEFUNCTION = ".hdbtablefunction";
-    /**
-     * File extension for *.hdbschema files
-     */
-    public static final String FILE_EXTENSION_HDBSCHEMA = ".hdbschema";
-    /**
-     * File extension for *.hdi files
-     */
-    public static final String FILE_EXTENSION_HDI = ".hdi";
+  /** File extension for *.hdbtable files */
+  public static final String FILE_EXTENSION_TABLE = ".hdbtable";
+  /** File extension for *.hdbview files */
+  public static final String FILE_EXTENSION_VIEW = ".hdbview";
+  /** File extension for *.hdbdd files */
+  public static final String FILE_EXTENSION_ENTITIES = ".hdbdd";
+  /** File extension for *.calculationview files */
+  public static final String FILE_EXTENSION_CALCULATION_VIEW = ".calculationview";
+  /** File extension for *.hdbcalculationview files */
+  public static final String FILE_EXTENSION_HDBCALCULATION_VIEW = ".hdbcalculationview";
+  /** File extension for *.hdbprocedure files */
+  public static final String FILE_EXTENSION_HDBPROCEDURE = ".hdbprocedure";
+  /** File extension for *.hdbtablefunction files */
+  public static final String FILE_EXTENSION_HDBTABLEFUNCTION = ".hdbtablefunction";
+  /** File extension for *.hdbschema files */
+  public static final String FILE_EXTENSION_HDBSCHEMA = ".hdbschema";
+  /** File extension for *.hdi files */
+  public static final String FILE_EXTENSION_HDI = ".hdi";
```

Comment:
This formatting looks strange.
In editconfig file there is ij_java_doc_do_not_wrap_if_one_line = false
which means that the comment should be not formatted in one line, as it is.

Reply:
+1, could you revert the comments formatting to be on multiple lines?